doc/tutorials/run_kata_containers.rst

```diff
@@ -58,41 +59,88 @@ Install Kata Containers
 ***********************
 
 Kata Containers provide a variety of installation methods, this guide uses
-:command:`kata-manager` to automate the Kata Containers installation procedure.
+`kata-deploy <https://github.com/kata-containers/packaging/tree/master/kata-deploy>`_
```

Comment:
Haven't tried kata-deploy, but what is the motivation to switch from kata-manager to kata-deploy?

Reply:
There are no builds for Ubuntu 20.04 (which I am using)... so `kata-manager` fails. See
* https://github.com/kata-containers/ci/issues/277
* https://github.com/kata-containers/tests/issues/1691

That's when I decided to use `kata-deploy`, which is rather straightforward to use and can work with both Ubuntu 18.04 and 20.04 (and I guess pretty much any other Linux distro too).